File: Python/3301.py
n = input().split()
for c, i in enumerate(n):
    n[c] = int(i)

a = int(n[0])
b = int(n[1])
c = int(n[2])


if max(n) == a and min(n) == b:
    print('luisinho')
elif max(n) == a and min(n) == c:
    print('zezinho')
    
elif max(n) == b and min(n) == a:
    print('luisinho')
elif max(n) == b and min(n) == c:
    print('huguinho')

elif max(n) == c and min(n) == a:
    print('zezinho')
elif max(n) == c and min(n) == b:
    print('huguinho')

File: C/3301.c
#include <stdio.h>
int main() {

    int a,b,c;
    scanf("%d%d%d", &a,&b,&c);
    if(a > b && a < c){
        printf("huguinho\n");
    }else if(a < b && a > c){
        printf("huguinho\n");
    }else if(b > a && b < c){
        printf("zezinho\n");
    }else if(b < a && b > c){
        printf("zezinho\n");
    }else{
        printf("luisinho\n");
    }
}
